Mathlib/Topology/Algebra/ClosedSubgroup.lean

```diff
@@ -0,0 +1,128 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nailin Guan
```

Comment:
Is this correct? Usually both names are the same, when a new file is started.

Reply:
Sorry that I just copy the file from my repo, maybe I should change it.